src/docs/product/session-replay/performance-overhead.mdx

```diff
@@ -36,25 +56,11 @@ The benchmarks were run on an Apple M1 MacBook Pro against a remote preview serv
 The benchmark test was a strenuous recording scenario (the Discover data table is one of our most complex, in regards to DOM nodes and mutations). A simpler scenario consisting of navigation to four different "Settings" pages was also ran and produced an increase of ~100 ms of total JS blocking time.
 
 <Note>
-  There is a known performance issue that happens when replays with many DOM
+  There's a known performance issue that happens when replays with many DOM
```

Comment:
Pedantic i know. There's a few more spot on this page where the apostrophe is  `'` and not `’`.

```suggestion
  There’s a known performance issue that happens when replays with many DOM
```

Reply:
No, totally appreciate it! Details matter :)